frontend/test/metabase/scenarios/dashboard/subscriptions.cy.spec.js

```diff
@@ -76,6 +76,26 @@ describe("scenarios > dashboard > subscriptions", () => {
           cy.findByRole("checkbox").should("have.attr", "aria-checked", "true");
         });
     });
+
+    it.skip("should not display 'null' day of the week (metabase#14405)", () => {
+      assignRecipient();
+      cy.findByText("To:").click();
+      cy.get(".AdminSelect")
+        .contains("Daily")
+        .click();
+      cy.findByText("Monthly").click();
+      cy.get(".AdminSelect")
+        .contains("First")
+        .click();
+      cy.findByText("15th (Midpoint)").click();
+      cy.get(".AdminSelect")
+        .contains("15th (Midpoint)")
+        .click();
+      cy.findByText("First").click();
+      clickButton("Done");
+      // Implicit assertion (word mustn't contain string "null")
+      cy.findByText(/^Emailed monthly on the first (?!null)/);
```

Comment:
Had to think about this. I figure instead of `Emailed monthly on the first null at 8:00 AM` it should say `Emailed monthly on the first at 8:00 AM`, so this is okay. Maybe it should say "first of the month", in which case this is still okay.

Reply:
It works even for `/^Emailed monthly (?!null)/` because it looks forward for `"null"`. I didn't want to be too specific, because I didn't know if the case will be "the first of the month" or something else, but in any case - "null" shouldn't be a part of this string.